app/javascript/modules/listingDetails/ListingDetailsImageCard.tsx

```diff
@@ -19,16 +28,35 @@ export const ListingDetailsImageCard = ({ listing }: ListingDetailsImageCardProp
             : undefined
         }
       />
-      <div className="p-3">
-        <p className="font-alt-sans uppercase tracking-widest text-sm font-semibold">
-          {getListingAddressString(listing)}
-        </p>
-        <p className="text-gray-700 text-base">{listing.Developer}</p>
-        <p className="text-xs">
-          <a href={"/"} target="_blank" aria-label="Opens in new window">
-            {t("label.viewOnMap")}
-          </a>
+      <div className="flex flex-col md:items-start md:text-left p-3 text-center">
+        <h1 className="font-sans font-semibold text-3xl">{listing.Name}</h1>
+        <p className="my-1">
+          <ListingAddress listing={listing} />
         </p>
+        <div className="flex flex-col items-center md:flex-row md:justify-between md:w-full">
+          <div>
+            <p className="text-gray-700">{listing.Developer}</p>
+            <p className="my-2">
+              <a
+                href={`https://www.google.com/maps/place/${encodeURIComponent(listingAddress)}`}
+                target="_blank"
+                aria-label={t("label.opensInNewWindow", { linkText: t("label.viewOnMap") })}
+              >
+                {t("label.viewOnMap")}
+              </a>
+            </p>
+          </div>
+          <div>
+            <a href={`${getShareListingPath()}/${listing.listingID}`} target="_blank">
+              <img
+                alt={t("label.share")}
+                onMouseEnter={(e) => (e.currentTarget.src = shareButtonSelected)}
+                onMouseLeave={(e) => (e.currentTarget.src = shareButton)}
```

Comment:
we should be able to accomplish this behavior (I think, not _positive_ an `img` tag won't act funky) while also getting similar interaction w/ keyboard users by doing 

```suggestion
                onFocus={(e) => (e.currentTarget.src = shareButtonSelected)}
                onBlur={(e) => (e.currentTarget.src = shareButton)}
```
you might also need to add `tabindex="0"` tho it already appears to be tab-able, so maybe unnecessary

Reply:
@jacksongonzales i updated the event handling so the image should change on hover and also when tabbing to the link